### PR TITLE
products: opensuse: Do not run the console tests on desktop scenarios

### PR DESCRIPTION
**Pull request by hwoarang**
There is no good reason to duplicate the console tests on a desktop
environment so only run them in textmode scenarios. This simplifies
the DE scenarios and allows us to introduce a new scenarios for
headless environments.

This also means that the current ```extra_tests_on_(gnome|kde|xfce)``` on openSUSE's openQA will not run the console tests anymore so a new scenario has to be added which will essentially set ```DESKTOP=textmode EXTRATEST=1```

### CODE REVIEW

**Comment by sysrich** [comment]
Is there any good reason to NOT run these tests on desktop scenarios?

**Comment by okurz** [comment]
I guess the idea was to save time and ressources. What should be considered is that running the console tests is "cheap" by comparison, e.g. to an installation. So normally we prefer to run more tests in differing scenarios even though some might already be seen as being covered. We can still benefit from running the same scenarios more often (to catch sporadic issues) or uncover weird interrelated issues where any "console test module" might fail only in a desktop environment.

TL;DR: keep code as is and close PR unless you had a different use case in mind ;-)

**Comment by nilxam** [comment]
the reason is:

[16:18] <hwoarang> good morning
[16:20] <hwoarang> do you think it would be useful to add a new openqa scenario for textmode + extra_tests similar to extra_tests_on_(gnome|kde|xfce)?
[16:22] <DimStar> hwoarang: if you have a good set of tests to run in this scenario, certainly
[16:24] <hwoarang> DimStar: most of the extra_tests do not actually need UI (in theory) so it might be useful to see whether that's true or not ;p
[16:28] <maxlin> actually if you see current extra_test not really had test UI stuff https://openqa.opensuse.org/tests/221557 :)
[16:29] <maxlin> I mean GUI
[16:30] <hwoarang> yep
[16:33] <maxlin> so I wonder extra_test_(textmode/console) might actually meaning move them to another test then or another test did same thing :p
[16:37] <coolo> maxlin: I'd go for an extra test
[16:38] <coolo> maxlin: I'd go for extra_tests_textmode and then limit _(kde|gnome|xfce) to those that make a difference
[16:38] <maxlin> ^ I guess you mean *avoding duplicate tests*?
[16:38] <maxlin> ah, yes
[16:39] <maxlin> sorry, typing slowly
[16:40] <maxlin> yes, I agree move them to extra_test_textmode and don't do duplicate tests in each extra_tests_xxx

and what is the reason you think we needs the duplicated extra console tests in the each extra_test_desktop?

**Comment by coolo** [comment]
you can leave it as is for sle - but for opensuse we just have too many desktops and don't care for "catch sporadic fails"

**Comment by coolo** [comment]
and @hwoarang: please add a "as discussed on IRC" next time :)

**Comment by nilxam** [comment]
testsuite/job-group is updated on o3.

**extra_tests_on_textmode**
BOOTFROM=c
BOOT_HDD_IMAGE=1
DESKTOP=textmode
EXTRATEST=1
HDD_1=%DISTRI%-%VERSION%-%ARCH%-%BUILD%-%DESKTOP%@%MACHINE%.qcow2
START_AFTER_TEST=textmode-image
SYSAUTHTEST=1

**Comment by okurz** [comment]
On Tuesday 12 July 2016 21:42:20 Stephan Kulow wrote:
> and @hwoarang: please add a "as discussed on IRC" next time :)

good idea. Or better describe the reasoning behind the change, i.e. what is 
the benefit. Thanks.

**Comment by hwoarang** [comment]
@okurz @coolo sorry guys. Will do next time. Thanks for merging this.

**Comment by okurz** [comment]
@hwoarang @DimStar77 @nilxam In the current state https://openqa.opensuse.org/tests/overview?distri=opensuse&version=Tumbleweed&groupid=1&test=yast2_ui_gnome does not make any sense anymore and yast2 ui tests are not called at all, see #1560 for a potential fix. Please discuss there.
